.github/workflows/ci.yml

```
@@ -78,9 +78,55 @@ jobs:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    needs: build
+    env:
+      DB_PORT: 5432
+      DB_USER: postgres
+      DB_PASS: postgres
+      DB_NAME: postgres
+      DB_HOST: localhost
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)